tests/test-sessions/test-hazelcast-sessions/src/test/java/org/eclipse/jetty/hazelcast/session/client/ClientModifyMaxInactiveIntervalTest.java

```diff
@@ -30,13 +36,36 @@
     extends AbstractModifyMaxInactiveIntervalTest
 {
 
+    private static final String MAP_NAME = "jetty_foo_session";
+
+    private HazelcastInstance hazelcastInstance;
+
+    @Before
+    public void startHazelcast()
+        throws Exception
+    {
+        Config config = new Config().addMapConfig( new MapConfig().setName( MAP_NAME ) ) //
+            .setInstanceName( "beer" );
```

Comment:
Also, should the names of the session maps/instances  use the name of the test?

Reply:
better changes in the next commit to use random names to avoid conflict with tests running in parallel